packages/Ribbon/Ribbon.md

```diff
@@ -0,0 +1,5 @@
+```jsx
+<Ribbon ribbonCopy="Telus Ribbon" />
+```
+
+- `Ribbon` is a fixed width element and will take a string as a prop to display its content.
```

Comment:
This is not accurate, the component is not fixed width. The width adjusts to the content.